tests/system/fim/common_tasks/verify_alerts_elasticsearch.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+from elasticsearch import Elasticsearch
+
+def setElasticsearch( ElasticIP):
```

Comment:
Please add the proper comments to these methods and ensure this script is `Flake8` compliant.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/482/commits/3098c93c91f79f9c2a854229fe9c38ed0162ac47

```
flake8.checker            MainProcess     66 INFO     Making checkers
flake8.checker            MainProcess     68 INFO     Checking 1 files
flake8.main.application   MainProcess     80 INFO     Finished running
flake8.main.application   MainProcess     80 INFO     Reporting errors
flake8.main.application   MainProcess     80 INFO     Found a total of 0 violations and reported 0
```